jme3-android-examples/jme3-android-tests/app/src/androidTest/java/com/scrappers/myapplication/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.scrappers.myapplication;
```

Comment:
why the package name is changed to `com.scrappers`?

Reply:
Oh, i did make sure that all package names are assigned to ```jme3``` but seems i have forgot to refractor the testClasses